src/components/FrameworkLinks.tsx

```diff
@@ -71,146 +71,70 @@ const FrameworkLinks = forwardRef(
-              >
-                Fastify
-              </Button>
-            )}
-            {(!exclude || !exclude.includes("nest-js")) && (
-              <Button
-                as="link"
-                size="lg"
-                href={`${path}?f=nest-js`}
-                decoratorLeft={<IconNestJs />}
-              >
-                NestJS
-              </Button>
-            )}
-            {(!exclude || !exclude.includes("next-js")) && (
-              <Button
-                as="link"
-                size="lg"
-                href={`${path}?f=next-js`}
-                decoratorLeft={<IconNextJs />}
-              >
-                Next.js
-              </Button>
-            )}
-            {(!exclude || !exclude.includes("node-js")) && (
-              <Button
-                as="link"
-                size="lg"
-                href={`${path}?f=node-js`}
-                decoratorLeft={<IconNodeJs />}
-              >
-                Node.js
-              </Button>
-            )}
-            {(!exclude || !exclude.includes("node-js-express")) && (
-              <Button
-                as="link"
-                size="lg"
-                href={`${path}?f=node-js-express`}
-                decoratorLeft={<IconNodeJs />}
-              >
-                Node.js + Express
-              </Button>
-            )}
-            {(!exclude || !exclude.includes("node-js-hono")) && (
-              <Button
-                as="link"
-                size="lg"
-                href={`${path}?f=node-js-hono`}
-                decoratorLeft={<IconNodeJs />}
-              >
-                Node.js + Hono
-              </Button>
-            )}
-            {(!exclude || !exclude.includes("nuxt")) && (
-              <Button
-                as="link"
-                size="lg"
-                href={`${path}?f=nuxt`}
-                decoratorLeft={<IconNuxt />}
-              >
-                Nuxt
-              </Button>
-            )}
-            {(!exclude || !exclude.includes("react-router")) && (
-              <Button
-                as="link"
-                decoratorLeft={<IconReactRouter />}
-                href={`${path}?f=react-router`}
-                size="lg"
-              >
-                React Router
-              </Button>
-            )}
-            {(!exclude || !exclude.includes("remix")) && (
-              <Button
-                as="link"
-                size="lg"
-                href={`${path}?f=remix`}
-                decoratorLeft={<IconRemix />}
-              >
-                Remix
-              </Button>
-            )}
-            {(!exclude || !exclude.includes("sveltekit")) && (
-              <Button
-                as="link"
-                size="lg"
-                href={`${path}?f=sveltekit`}
-                decoratorLeft={<IconSvelteKit />}
-              >
-                SvelteKit
-              </Button>
-            )}
+            {frameworks.map((f) => {
```

Comment:
This refactor feels like a different idea than tweaking the logo

Reply:
I might have scope-creeped a little, would you like me to split this into a separate PR?

Reply:
Its ok, we'll keep it between us 🤫